### DISCUSSION TITLE
No module named 'intensity_normalization.typing'

### DISCUSSION THREAD

**Discussion by bax24** [jcreinhold/intensity-normalization#62] [Q&A] [answered]
Hello,

I have installed all the required packages and I still get this error when running on jupyter notebook 

"ModuleNotFoundError: No module named 'intensity_normalization.typing'"

Could you help me ?

thanks a lot in advance

**Comment by jcreinhold** [comment, marked as answer]
What version of the code are you running and what version of Python? I suspect you're using Python != 3.9 which is, currently, the only version of Python that's supported for the latest, maintained version of the package.

**Comment by bax24** [comment]
Yes you are right I created a new environment with python 3.9 and it works !

Thanks a lot !
